crates/xline/src/storage/kvwatcher.rs

```diff
@@ -552,6 +552,19 @@ pub(crate) struct WatchEvent {
     compacted: bool,
 }
 
+impl std::fmt::Debug for WatchEvent {
+    #[allow(clippy::arithmetic_side_effects)]
```

Comment:
If it's affected by the macro, please fix it in the macro.

Reply:
Done